### DISCUSSION TITLE
Welcome to White-Board Discussions!

### DISCUSSION THREAD

**Discussion by SujalXplores** [SujalXplores/White-Board#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SujalXplores** [comment]
Suggest me a feature.

I'm planning to upgrade this app to next level. 🎉

**Reply by vishal-rathod-07** [reply]
Hey @SujalShah3234 ,

    There are many things you can add option for draw different shape into the board.

**Comment by vishal-rathod-07** [comment]
You can also add image option so we can add images into the board.
